.github/workflows/release.yml

```diff
@@ -175,8 +175,8 @@ jobs:
 
       - name: Install pnpm
         uses: pnpm/action-setup@v3
-        # version is no longer being defined here, since we can automatically use the version
-        # defined in workers-sdk/package.json instead.
+        with:
+          version: 9.1.3
```

Comment:
Using `~9.1.3` or `^9.1.3` might delay the time when this will be broken next – or that syntax might be unsupported and just won't work, we can go ahead with this for now.